README.md

```diff
@@ -183,6 +183,9 @@ the following form (`[...]` denotes optional or conditional values),
  :fetcher [git|github|gitlab|bitbucket|bzr|hg|darcs|fossil|svn|cvs|wiki]
  [:url "<repo url>"]
  [:repo "github-gitlab-or-bitbucket-user/repo-name"]
+ [:commit "<ref>"]
```

Comment:
can you remove `:commit` and `:branch` here because they are kind of hacks that I actually wish we would remove. I don't want to advertise them. there are cases where we have made exceptions to this rule but in general we avoid these things like the plague. If there is ever a time that no recipe uses either I will remove the functionality.